packages/EasyDecision/src/Interfaces/MappingProviderInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyDecision\Interfaces;
+
+interface MappingProviderInterface
+{
+    /**
+     * Returns decision type.
```

Comment:
Please remove this PHPDoc, as in this package it's preferred to add them only when some annotations are needed.

Reply:
Done.